src/scripts/components/board/LeftFields.js

```diff
@@ -16,29 +16,29 @@ class LeftFields extends React.Component {
   render() {
     return (
       <div className="left-box">
-        <div className={'card-left owner-' + this.state.fields[39].owner} id="p39">
-          <div className="label">Stare Zoo</div>
+        <div className={'card-left owner-' + this.state.fields[9].owner} id="p9">
+          <div className="label">{/*Cmentarz Junikowo*/}</div>
         </div>
-        <div className={'card-left owner-' + this.state.fields[38].owner} id="p38">
-          <div className="label">Teatr nowy</div>
+        <div className={'card-left owner-' + this.state.fields[8].owner} id="p8">
+          <div className="label">{/*STadion Inea*/}</div>
         </div>
-        <div className={'card-left owner-' + this.state.fields[37].owner} id="p37">
+        <div className={'card-left owner-' + this.state.fields[7].owner} id="p7">
         </div>
-        <div className={'card-left owner-' + this.state.fields[36].owner} id="p36">
-          <div className="label">Palmiarnia</div>
+        <div className={'card-left owner-' + this.state.fields[6].owner} id="p6">
+          <div className="label">{/*Pixel*/}</div>
         </div>
-        <div className={'card-left owner-' + this.state.fields[35].owner} id="p35">
+        <div className={'card-left owner-' + this.state.fields[5].owner} id="p5">
+          <div className="label">{/*Pętla Tranwajowwa Grunwalt*/}</div>
         </div>
-        <div className={'card-left owner-' + this.state.fields[34].owner} id="p34">
-          <div className="label">Kino</div>
+        <div className={'card-left owner-' + this.state.fields[4].owner} id="p4">
         </div>
-        <div className={'card-left owner-' + this.state.fields[33].owner} id="p33">
-          <div className="label">Aquanet</div>
+        <div className={'card-left owner-' + this.state.fields[3].owner} id="p3">
+          <div className="label">{/*Kino Biedronka */}</div>
```

Comment:
There is better to comment element (not only text inside). It is for all elements like that